src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -95,7 +95,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                     items.AddRange(GetListHeaderItems(span));
                 }
 
-                if (token.Parent.Parent is DocumentationCommentTriviaSyntax)
+                if (token.Parent.Parent is DocumentationCommentTriviaSyntax ||
+                    (token.Parent.Parent.IsKind(SyntaxKind.XmlEmptyElement) && token.Parent.Parent.Parent is DocumentationCommentTriviaSyntax))
```

Comment:
This isn't quite right. Consider:
```C#
///<summary>
/// <r$$
///</summary>
```

In that case, the token's great-grandparent (`token.Parent.Parent.Parent`) is an XmlElementSyntax.